packages/autoprofile/package.json

```diff
@@ -56,7 +56,7 @@
     "semver": "^7.5.4"
   },
   "engines": {
-    "node": ">=14.0.0"
```

Comment:
Putting it here.
I noticed that core package.json does not have engines definition. Could u please add?

Reply:
added